notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -4764,6 +4764,51 @@ spouse murder [It turned out that Claire Wellington had murdered her husband Mik
 extramarital affair [It turned out that Claire Wellington killed her husband, Mike, upon catching him cheating on her with a 21 year old woman.]
 
 
+movie: Post Impact (2004)
+=========================
+
+:: Title
+Post Impact
+
+:: Description
+Post Impact is a 2004 disaster film, written and directed by Christoph Schrewe
+and stars Dean Cain, Bettina Zimmermann, Joanna Taylor, Nigel Bennett, and
+Hanns Zischler.
+
+Synopsis: An army captain is forced to leave his family behind during a
+massive impact event.
+
+:: Date
+2004-06-12
+
+:: References
+https://en.wikipedia.org/wiki/Post_Impact
+
+:: Ratings
+1 <paul>
+
+:: Choice Themes
+asteroid impact [The title refers to the aftermath of a large cometary fragment impacting the Earth, western Russia specifically. A new geopolitical order was established within three years of the catastrophic impact, which had kick started the world into a new ice age.]
+
+:: Major Themes
+remorse [Tom deeply regretted having been forced to leave his family behind in the lead up to the comet impact. He was motivated thereafter by a need to go back to find out what had become of them.]
```

Comment:
Just for the record: I think it's wrong to write "regret having been forced", at least in the sense of regret=remorse, with the definition we use for "remorse". He might regret having let himself be forced, perhaps. Or simply "he felt guilty over leaving ..."

Reply:
A point well taken. I revised to motivation to read as "Tom deeply regretted having let himself be forced to leave his family behind...".